packages/components/workbench/src/Workbench.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Workbench'
+type: 'component'
+status: 'stable'
+slug: /components/workbench/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/workbench'
+storybook: 'https://f36-storybook.contentful.com/?path=/docs/components-workbench--basic'
+typescript: ./Workbench.tsx,./WorkbenchHeader/WorkbenchHeader.tsx,./WorkbenchContent/WorkbenchContent.tsx,./WorkbenchSidebar/WorkbenchSidebar.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+An app shell that structures regions of content and interaction.
+The workbench assembles the outer app shell which defines regions to structure content and interactions.
+
+## Table of contents
```

Comment:
we need to mention that this package is not included by default and the components can be imported from another package